### ISSUE TITLE
哪一步的权重文件是最好的

### ISSUE THREAD

**Issue by xjushengjie** [H-EmbodVis/TurboVLA#12]
博主，我看了我们这篇论文有点疑惑，能不能解答一下呢。在训练是我看我们方法在libero上训练了8w步，我们是怎么判断那一步保存的权重文件是最好的呢？只看loss的指标吗？是越低越好吗？

**Comment by zhoutao-sdu2025** [comment]
> 博主，我看了我们这篇论文有点疑惑，能不能解答一下呢。在训练是我看我们方法在libero上训练了8w步，我们是怎么判断那一步保存的权重文件是最好的呢？只看loss的指标吗？是越低越好吗？

Hello, we have conducted experiments on LIBERO‑Long with 80k training steps using A100 GPUs and a gradient accumulation factor of 2. We also evaluated models trained for 20k, 40k, and 80k steps. The best performance of 88.8% success rate is achieved at 40k steps.

**Comment by xjushengjie** [comment]
> > 博主，我看了我们这篇论文有点疑惑，能不能解答一下呢。在训练是我看我们方法在libero上训练了8w步，我们是怎么判断那一步保存的权重文件是最好的呢？只看loss的指标吗？是越低越好吗？
> 
> Hello, we have conducted experiments on LIBERO‑Long with 80k training steps using A100 GPUs and a gradient accumulation factor of 2. We also evaluated models trained for 20k, 40k, and 80k steps. The best performance of 88.8% success rate is achieved at 40k steps.

libero-long就是libero-10吗？你的global batchsize是多少啊？我浮现这精度也差点，不是那么完美

**Comment by zhoutao-sdu2025** [comment]
> > > 博主，我看了我们这篇论文有点疑惑，能不能解答一下呢。在训练是我看我们方法在libero上训练了8w步，我们是怎么判断那一步保存的权重文件是最好的呢？只看loss的指标吗？是越低越好吗？
> > 
> > 
> > Hello, we have conducted experiments on LIBERO‑Long with 80k training steps using A100 GPUs and a gradient accumulation factor of 2. We also evaluated models trained for 20k, 40k, and 80k steps. The best performance of 88.8% success rate is achieved at 40k steps.
> 
> libero-long就是libero-10吗？你的global batchsize是多少啊？我浮现这精度也差点，不是那么完美

单卡是64，两张A100，梯度积累为2，一共256和作者保持一致
你的准确率有多少，能否留个联系方式交流下，哈哈哈

**Comment by xjushengjie** [comment]
> > > > 博主，我看了我们这篇论文有点疑惑，能不能解答一下呢。在训练是我看我们方法在libero上训练了8w步，我们是怎么判断那一步保存的权重文件是最好的呢？只看loss的指标吗？是越低越好吗？
> > > 
> > > 
> > > Hello, we have conducted experiments on LIBERO‑Long with 80k training steps using A100 GPUs and a gradient accumulation factor of 2. We also evaluated models trained for 20k, 40k, and 80k steps. The best performance of 88.8% success rate is achieved at 40k steps.
> > 
> > 
> > libero-long就是libero-10吗？你的global batchsize是多少啊？我浮现这精度也差点，不是那么完美
> 
> 单卡是64，两张A100，梯度积累为2，一共256和作者保持一致 你的准确率有多少，能否留个联系方式交流下，哈哈哈
我在4090上按照16，4，4复现的，结果还差点。
我自己浮现的结果
                   | goal | long | object | spatial
自己训练的 | 96 | 89 | 98.8 | 97.6
官方发布 | 96.2 | 93.6 | 99.2 | 98.2
你可以加我好友：1358928376

**Comment by zhoutao-sdu2025** [comment]
> > > > > 博主，我看了我们这篇论文有点疑惑，能不能解答一下呢。在训练是我看我们方法在libero上训练了8w步，我们是怎么判断那一步保存的权重文件是最好的呢？只看loss的指标吗？是越低越好吗？
> > > > 
> > > > 
> > > > Hello, we have conducted experiments on LIBERO‑Long with 80k training steps using A100 GPUs and a gradient accumulation factor of 2. We also evaluated models trained for 20k, 40k, and 80k steps. The best performance of 88.8% success rate is achieved at 40k steps.
> > > 
> > > 
> > > libero-long就是libero-10吗？你的global batchsize是多少啊？我浮现这精度也差点，不是那么完美
> > 
> > 
> > 单卡是64，两张A100，梯度积累为2，一共256和作者保持一致 你的准确率有多少，能否留个联系方式交流下，哈哈哈
> > 我在4090上按照16，4，4复现的，结果还差点。
> > 我自己浮现的结果
> >                    | goal | long | object | spatial
> > 自己训练的 | 96 | 89 | 98.8 | 97.6
> > 官方发布 | 96.2 | 93.6 | 99.2 | 98.2
> > 你可以加我好友：1358928376

那咱两其实差不多，你可以加我，你少一位号码，15056631495，咱两交流一下

**Comment by IMMORTAALLL** [comment]
@xjushengjie @zhoutao-sdu2025 Thank you for your interest in TurboVLA and for sharing your reproduction results.
To clarify the checkpoint-selection procedure, we do not select checkpoints based solely on training loss. Checkpoints are saved at regular intervals and compared using downstream LIBERO success rates.
For reproducing the released results, we recommend using an effective global batch size of 128 and evaluating the EMA weights through ema_model_state_dict when available. The latest commit makes this recommended setup explicit.
The newly released single mixed-training checkpoint is the 34k-step EMA checkpoint from the joint training run.
Thank you again for helping us improve the release.
